src/components/contributors/index.js

```diff
@@ -11,10 +11,12 @@ const styles = StyleSheet.create({
   }
 });
 
-export default function Contributors() {
-  return (
+const Contributors = () => {
+  (
     <View style={styles.content}>
```

Comment:
Did you remove the `return` on purpose?

Reply:
> Did you remove the `return` on purpose?

I did but as CJ is showing I didn't do it quite right lol.

Reply:
Please remove curly braces to be able to leave off return. Applies to all other functions as well.

Reply:
Fixed incorrect curly brace inclusion.